src/content/reference/react-dom/components/option.md

```diff
@@ -4,12 +4,13 @@ title: "<option>"
 
 <Intro>
 
-The [built-in browser `<option>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) lets you render an option inside a [`<select>`](/reference/react-dom/components/select) box.
+[Komponen `<option>` bawaan peramban](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) memungkinkan Anda untuk me-*render* opsi di dalam kotak [`<select>`](/reference/react-dom/components/select).
+
 
 ```js
 <select>
-  <option value="someOption">Some option</option>
-  <option value="otherOption">Other option</option>
+  <option value="sebuahOpsi">Sebuah opsi</option>
```

Comment:
I think the value (`someOption` and `otherOption`) shouldn't be translated, it has the same level as a variable name, let's keep it as it is. Only the UI text can be translated.

Reply:
what about value for this [FruitPicker component](https://github.com/reactjs/id.react.dev/pull/368/files#diff-f8e5a07bbbedc1f852d250d167b646f71fc5a5dcc8a76534c437db9c517df257R65)?

Reply:
@aroyan I think we should roll back only the value to the original translation on that part.

Reply:
Rule of thumb:

- If it's for the code: don't translate
- If it's not for the code or just a content: can be translated